README.md

```diff
@@ -18,25 +18,33 @@ authenticated.
 1. Download and decompress the latest plugin binary from the Releases tab on
 GitHub. Alternatively you can compile the plugin from source.
 
-1. Move the compiled plugin into Vault's configured `plugin_directory`:
+2. Move the compiled plugin into Vault's configured `plugin_directory`:
 
   ```sh
-  $ mv vault-auth-example /etc/vault/plugins/vault-auth-example
+  $ mv vault-auth-plugin-example /etc/vault/plugins/vault-auth-plugin-example
   ```
 
-1. Calculate the SHA256 of the plugin and register it in Vault's plugin catalog.
+3. If you are running Vault on Linux as a non-root user, you should enable the plugin
```

Comment:
I don't think this had to do with Linux and mlock, it is more about if your Vault server is running with mlock on a supported platform.